rented-properties/src/main/java/org/egov/cpt/service/DuplicateCopyService.java

```diff
@@ -109,8 +110,12 @@
 
 	private List<DuplicateCopy> getApplication(DuplicateCopySearchCriteria criteria, RequestInfo requestInfo) {
 		List<DuplicateCopy> applications = repository.getDuplicateCopyProperties(criteria);
-		if (applications.isEmpty())
-			return Collections.emptyList();
+		if (applications.isEmpty()){
+			if(requestInfo.getUserInfo().getType().equalsIgnoreCase(PTConstants.ROLE_CITIZEN))
+				throw new CustomException("INVALID ACCESS", "You can not access this application.");
+			else
+				return Collections.emptyList();
+		}
```

Comment:
This will cause new citizens to see the error in MyApplications also

Reply:
We should throw this error if the user is citizen and they are trying to fetch by specific application id

Reply:
Citizen can access only his applications in My Applications